sync/gh.py

```diff
@@ -550,7 +550,7 @@ def merge_sha(self, pr_id: int) -> Optional[str]:
 
     def merge_pull(self, pr_id: int) -> str:
         pr = self.get_pull(pr_id)
-        if self.is_mergeable:
+        if self.is_mergeable is not None:
```

Comment:
Hmm what's the type here? From the base class it seems like it ought to be a bool, in which case this changes the behaviour for `False`. So I'm clearly missing something…

Reply:
`is_mergeable` is actually a function, so the error was `error: Function "Callable[[int], bool]" could always be true in boolean context`. But looking at the code again, maybe we actually want here to call it?

Reply:
Ah, so it was hilariously broken :) Yes, we want to do `self.is_,mergeable(pr_id)`. Not sure how I missed that.